src/subscription_manager/managercli.py

```diff
@@ -606,7 +607,6 @@ def __init__(self, name, shortdesc=None, primary=False, attr=None, commands=('se
 
     def _get_synced_store(self):
         try:
-            from syspurpose.files import SyncedStore
```

Comment:
It doesn't make sense to remove this import statement and not remove `try-except ImportError`. I would keep this method as is, but I would move the method `_get_synced_store()` to `CliCommand` and then this method could be used in `RegisterCommand` subclass of `CliCommand` too.